src/docker_hub.ml

```diff
@@ -350,15 +358,19 @@ module Config = struct
     Json.pp fmt json
 end
 
+external reraise : exn -> 'a = "%reraise"
```

Comment:
This might be nice to have in Miou (or in the stdlib but since the stdlib doesn't have it currently it would be nice to have anyway for backward compatibility)